### PR TITLE
Add pytest-tornasync recipe

### PR DESCRIPTION
**Pull request by lincrosenbach**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pytest-tornasync/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - setuptools
+  run:
+    - pytest >=3.0
+    - python >=3.5
+    - tornado >=5.0
+
+test:
+  imports:
+    - pytest_tornasync
+  requires:
```
Comment (nicoddemus) [inline recipes/pytest-tornasync/meta.yaml]: You don't need this section, those dependencies are already declared in `run`. 👍

File: recipes/pytest-tornasync/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  license_file: {{ RECIPE_DIR }}/LICENSE
+  summary: py.test plugin for testing Python 3.5+ Tornado code
+  dev_url: https://github.com/eukaryote/pytest-tornasync
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - icandeia
+    - kfasolin
+    - nicoddemus
```
Comment (nicoddemus) [inline recipes/pytest-tornasync/meta.yaml]: 👍 in being maintainer

File: recipes/pytest-tornasync/meta.yaml (hunk 1 above)
Comment (lincrosenbach) [reply recipes/pytest-tornasync/meta.yaml]: Nice. I´ll remove them.

File: recipes/pytest-tornasync/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  license_family: MIT
+  license_file: {{ RECIPE_DIR }}/LICENSE
+  summary: py.test plugin for testing Python 3.5+ Tornado code
+  dev_url: https://github.com/eukaryote/pytest-tornasync
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - icandeia
+    - kfasolin
```
Comment (kfasolin) [inline recipes/pytest-tornasync/meta.yaml]: I'm ok being a maintainer

File: recipes/pytest-tornasync/meta.yaml (hunk 1 above)
Comment (lincrosenbach) [reply recipes/pytest-tornasync/meta.yaml]: Done.

File: recipes/pytest-tornasync/meta.yaml (hunk 3 above)
Comment (igorcandeia) [reply recipes/pytest-tornasync/meta.yaml]: I'm here?

**Comment by igorcandeia** [comment]
> Hi! This is the friendly automated conda-forge-linting service.
> 
> I just wanted to let you know that I linted all conda-recipes in your PR (`recipes/pytest-tornasync`) and found it was in an excellent condition.

I'm here now.

File: recipes/pytest-tornasync/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: {{ RECIPE_DIR }}/LICENSE
+  summary: py.test plugin for testing Python 3.5+ Tornado code
+  dev_url: https://github.com/eukaryote/pytest-tornasync
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - igorcandeia
```
Comment (igorcandeia) [inline recipes/pytest-tornasync/meta.yaml]: I'm here!

Comment (lincrosenbach) [reply recipes/pytest-tornasync/meta.yaml]: Are you ok to be a maintainer?

Comment (igorcandeia) [reply recipes/pytest-tornasync/meta.yaml]: Yes I am!

File: recipes/pytest-tornasync/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+test:
+  imports:
+    - pytest_tornasync
+
+about:
+  home: https://github.com/eukaryote/pytest-tornasync
+  license: MIT
+  license_family: MIT
+  license_file: {{ RECIPE_DIR }}/LICENSE
```
Comment (marcelotrevisani) [inline recipes/pytest-tornasync/meta.yaml]: You don't need this Jinja variable anymore. Please, remove it in the feedstock

Comment (marcelotrevisani) [reply recipes/pytest-tornasync/meta.yaml]: Let it just ``LICENSE`` and it is fine

Comment (marcelotrevisani) [reply recipes/pytest-tornasync/meta.yaml]: And please create an issue in pytest-tornasync to package the license as well

Comment (lincrosenbach) [reply recipes/pytest-tornasync/meta.yaml]: > And please create an issue in pytest-tornasync to package the license as well

The issue is already there. Thanks

Comment (lincrosenbach) [reply recipes/pytest-tornasync/meta.yaml]: > You don't need this Jinja variable anymore. Please, remove it in the feedstock

Nice!

Comment (lincrosenbach) [reply recipes/pytest-tornasync/meta.yaml]: > > You don't need this Jinja variable anymore. Please, remove it in the feedstock
> 
> Nice!

I will change it on feedstock! Thanks

**Pull request merged**